cmd/dfdaemon/app/config.go

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright The Dragonfly Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package app
+
+import (
+	"github.com/dragonflyoss/Dragonfly/pkg/printer"
+
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+	"gopkg.in/yaml.v2"
+)
+
+func init() {
+	configRootCommand := &cobra.Command{
+		Use:   "config",
+		Short: "Manage configuration of dfdaemon",
+		Args:  cobra.NoArgs,
+	}
+
+	defaultDFDaemonConfigPrintCommand := &cobra.Command{
```

Comment:
Maybe it's not necessary to specify `dfdaemon` in the variable name `defaultDFDaemonConfigPrintCommand` and it's enough with `defaultConfigPrintCommand ` because it's only for `dfademon`. WDYT?